docs/changelogs/CHANGELOG-2.25.md

```diff
@@ -0,0 +1,232 @@
+    - Some of the values from earlier `values.yaml` are now incompatible with latest version. They are removed in the `values.yaml` in the current chart. But if you had copied the original values.yaml to customize it further, you may see that `kubermatic-installer` will detect such incompatible options and churn out errors and explain that action that needs to be taken.
+    - The memcached-* charts are now subcharts of cortex chart so if you provided configuration for `memcached-*` blocks in your `values.yaml` for user-mla, you must move them under `cortex:` block.
+
+### Highlights
+- EE: Add KubeVirt ([#12851](https://github.com/kubermatic/kubermatic/pull/12851))
+- Upstream Documentation and SourceURLs can be added to ApplicationDefinitions ([#13019](https://github.com/kubermatic/kubermatic/pull/13019))
+- EE: Add k8sgpt operator to the Default Application Catalogue ([#13025](https://github.com/kubermatic/kubermatic/pull/13025))
+- Add K8sGPT to the Webshell ([#6501](https://github.com/kubermatic/dashboard/pull/6501))
+- Add new feature to create, restore and schedule backups for user cluster namespaces ([#6296](https://github.com/kubermatic/dashboard/pull/6296))
+- Add new page to manage backup storage location for the cluster backup feature ([#6478](https://github.com/kubermatic/dashboard/pull/6478))
+- Support for downloading backups from the UI ([#6521](https://github.com/kubermatic/dashboard/pull/6521))
+- Add support for Edge provider ([#6502](https://github.com/kubermatic/dashboard/pull/6502))
+- Display comments in application values ([#6510](https://github.com/kubermatic/dashboard/pull/6510))
+- Add Support for Kubernetes 1.29 ([#12936](https://github.com/kubermatic/kubermatic/pull/12936))
+
+
+### API Changes
+
+- Add the edge cloud provider ([#13018](https://github.com/kubermatic/kubermatic/pull/13018))
+- EtcdStatefulSetSettings: Add nodeSelector option to let etcd pods only run on specific nodes ([#12838](https://github.com/kubermatic/kubermatic/pull/12838))
+
+### Supported Kubernetes Versions
+
+- Add Support for Kubernetes 1.29 ([#12936](https://github.com/kubermatic/kubermatic/pull/12936))
+- Add support for Kubernetes v1.26.13, v1.27.10, v1.28.6, v1.29.1 ([#12981](https://github.com/kubermatic/kubermatic/pull/12981))
+- Update supported kubernetes versions ([#13079](https://github.com/kubermatic/kubermatic/pull/13079)):
+    - Add 1.29.2/1.28.7/1.27.11 to the list of supported Kubernetes releases. 
+    - Add 1.29 to the list of supported EKS versions. 
+    - Add 1.29 / remove 1.26 from the list of supported AKS versions
+- Remove support for Kubernetes 1.26 ([#13032](https://github.com/kubermatic/kubermatic/pull/13032))
+- Remove 1.25 from list of supported versions on AKS (EOL on January 14th) ([#12962](https://github.com/kubermatic/kubermatic/pull/12962))
+
+#### Supported Versions
+
+- 1.27.10
+- 1.27.11
+- 1.28.6
+- 1.28.7 (default for k8s)
+- 1.29.1
+- 1.29.2
+- 1.29 (default for EKS and AKS)
```

Comment:
No need to list this, historically this section is only for native user clusters. So this line can be removed, and the `(default for k8s)` note can also go.